modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMDashboards.testcase

```diff
@@ -99,6 +99,78 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-180526. Verify that the value of Claimed MDFs are displayed on MDF Dashboard"
+	@priority = 4
+	test MDFClaimedValueAppearInDashboard {
+		var budgetValue = 2000;
+
+		task ("Given the Channel Marketing Manager get the current amount of Claim MDF Value") {
+			PRMNavigator.openSitePage(pageName = "Home");
+
+			PRMDashboards.scrollViewDashboard(viewDashboard = "Market Development Funds");
+
+			var initialValue = PRMDashboards.getLegendValue(
+				chartName = "Request",
+				legendName = "Claimed");
+		}
+
+		task ("And a new MDF Request created and approved it") {
```

Comment:
Hi @marinaasant0s, please rename this task to `And a new MDF Request is created and approved`